src/main/java/com/github/netelli/TablesDAO.java

```diff
@@ -0,0 +1,39 @@
+package com.github.netelli;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public abstract class TablesDAO implements AutoCloseable {
+    final static Logger logger = Logger.getLogger(ProductsDAO.class);
+    public Connection connection;
```

Comment:
by using public you ruined encapsulation, should be protected because this class is abstract and intended to be extended